app/config/Constants.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+object Constants {
+  val BoxName = s"customs/transits##1.0##notificationUrl"
```

Comment:
The `customs/transits` part is determined by the API context, and the `notificationUrl` part is determined by the API subscription configuration, so the only remaining part that could be configurable is the version, which we decided to hardcode for now rather than extracting it from the Accept header I think?

Reply:
That's not to say we couldn't put this into the config file of course